keras_nlp/models/bert/bert_tasks.py

```diff
@@ -262,6 +264,7 @@ def from_preset(
         # Check if preset is backbone-only model
         if preset in Bert.presets:
             backbone = Bert.from_preset(preset, load_weights)
+            kwargs["dropout"] = Bert.presets[preset]["config"]["dropout"]
```

Comment:
Why do we need this line? This function lets the user set a separate `dropout` for the classifier and the you override it silently.

Reply:
In this if branch it reads the BERT presets instead of the classifier presets, so it does not have the `dropout` value. configured in classifier presets.